config.default.php

```diff
@@ -237,3 +237,10 @@
 
 // TOTP (2FA) issuer name
 define('TOTP_ISSUER', 'Kanboard');
+
+// default filter expression
+define('DEFAULT_FILTER', 'status:open');
+
+// an expression that will be added to the predefined filters
+// define('ADDITIONAL_FILTER', 'board_date:<=today');  // use this when plugin TaskBoardDate is installed
+define('ADDITIONAL_FILTER', '');  // use this otherwise
```

Comment:
Sorry for being unavailable for so long. I had (and actually still have) to focus my attention elsewhere...

I agree that core kanboard and plugins should not be mixed. So I guess, it's best to keep this idea as simple as possible, basically what the title says: "Define a default filter phrase in config.php", and let's ignore the board_date stuff for the moment.

I hope I will have more time in the near future to make some progress here...